src/screens/TrackingFlights/TrackingFlights.jsx

```diff
@@ -1,9 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import {
+  getItemFromLocalStorage,
+  saveItemInLocalStorage,
+} from 'utils/localStorage';
+import Offer from 'shared/components/Offer/Offer';
+import DeleteButton from 'shared/components/DeleteButton/DeleteButton';
 
 import './styles.scss';
 
 function TrackingFlights() {
-  return <div>tracking </div>;
-}
+  const [quotes, setQuotes] = useState([]);
+  const [qutoeDelete, setQuoteDelete] = useState(0);
+
+  useEffect(() => {
+    const quotes = getItemFromLocalStorage('quotes');
```

Comment:
Favourite quotes should be kept in store. You can populate initialState with `getItemFromLocalStorage('quotes')`

Reply:
Fixed